protocols/delegate-factory/test/Delegate-Factory-unit.js

```diff
@@ -78,6 +78,13 @@ contract('Delegate Factory Tests', async accounts => {
       )
     })
 
+    it('should not deploy a factory with indexer address 0x0', async () => {
```

Comment:
Overall having this check is unnecessary, as the contract is only deployed once by AirSwap. When deploying to Rinkeby and Mainnet, believe the interfaces are checked to match the function abi signature 

    require(address(_swapContract) != address(0),
      "SWAP_CONTRACT_REQUIRED");

    require(address(_indexerContract) != address(0),
      "INDEXER_CONTRACT_REQUIRED");